kickstarter/android-oss
# NT-1530: Add PLN Currency to the User's Settings Page

# 📲 What

Add the Polish currency to user settings to allow users to change their currency to the złoty

# 👀 See

![Screenshot_1601326365](https://user-images.githubusercontent.com/19390326/94486357-4f84b800-01ad-11eb-9bb8-cb3659112206.png) ![Screenshot_1601326372](https://user-images.githubusercontent.com/19390326/94486358-501d4e80-01ad-11eb-8b4e-721345c97b21.png) ![Screenshot_1601327057](https://user-images.githubusercontent.com/19390326/94486359-501d4e80-01ad-11eb-8547-db3c50e80cf4.png)

# 📋 QA

- Login to your account
- Tap overflow menu button in top left corner
- Select "Settings"
- Select "Account"
- In the "Currency" section, tap to reveal the menu. Select "zł Polish Złoty (PLN)"
- Confirm changes
- Navigate back to the home screen
- Tap any project, navigate to a reward
- Should see updated copy showing the Polish zł where applicable. 

# Story 📖

[NT-1530](https://kickstarter.atlassian.net/browse/NT-1530)
